### DISCUSSION TITLE
[Feature Request] Bitcoin RPC Proxy

### DISCUSSION THREAD

**Discussion by gcomte** [raspiblitz/raspiblitz#3958] [Feature / App Ideas]
The https://github.com/Kixunil/btc-rpc-proxy allows for finer-grained permission management for `bitcoind`.

It would also enable pruning for `bitcoind`, without abandoning applications that rely on non-pruned nodes like `LND` or `BTCPay Server`.

**Comment by openoms** [comment]
Fine grained permissions yes, but simulating an unpruned node does not work (yet).
See the discussion at https://github.com/cryptoadvance/specter-desktop/issues/722
